torch/_inductor/config.py

```diff
@@ -1223,16 +1223,6 @@ class trace:
     # SVG figure showing fx with fusion
     draw_orig_fx_graph = os.environ.get("INDUCTOR_ORIG_FX_SVG", "0") == "1"
 
-    # We draw our fx graphs with the "record" shape attribute by default.
-    # Sometimes, when the graph is very complex, we may hit dot errors like below:
-    #   "flat edge between adjacent nodes one of which has a record shape -
-    #    replace records with HTML-like labels"
-    # and thus fail to generate a graph. So, let's give the user an option
-    # to specify the shape attribute for the dot graph. For example, passing
-    # INDUCTOR_DOT_GRAPH_SHAPE_SVG = "none" would let us generate HTML-like lables
-    # to workaround the above failure.
-    dot_graph_shape = os.environ.get("INDUCTOR_DOT_GRAPH_SHAPE_SVG", None)
```

Comment:
Is there no longer a way to generate this ?

Reply:
My reasoning for removing the config flag is that it was originally added to resolve a bug that is now fixed by this PR. 

However, other shapes do work with html-like labels, so leaving it in wouldn't hurt anything. It's definitely a low impact config flag so if we're trying to simplify the config file, it would be a good one to remove.

Reply:
ah, didnt realize that this config was just for the bug.

Reply:
Have we ever documented it? If so, it probably should be kept around